src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +57,47 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block {
+    private static String block;
+
+    public Block(String content) {
+        this.block = content;
```

Comment:
Does it make sense? Assigning a variable to a static variable? Also `this` does not have a static variable `block`! `this` is an instance!